ui/src/app/views/workspace-edit/component.ts

```diff
@@ -238,8 +233,8 @@ export class WorkspaceEditComponent implements OnInit {
     }
     this.savingWorkspace = true;
     this.workspacesService.createWorkspace(this.workspace).subscribe(
-        () => {
-          this.navigateBack();
+        (workspace) => {
+          this.router.navigate(['workspace', workspace.namespace, workspace.id]);
```

Comment:
Also, once we have the tab URL thing, we may want to consider sending the user straight to the cohorts tab, rather than the notebooks on their first load.